### PR TITLE
Deactiate thankyou letters

### PR DESCRIPTION
**Pull request by quentingigon**
Added the field in the model and un-commented the filtering of invoices

### CODE REVIEW

**Comment by quentingigon** [comment]
other half is [HERE](https://github.com/CompassionCH/compassion-switzerland/pull/1099)

File: mobile_app_connector/models/account_invoice.py [hunk 1]
```diff
@@ -17,6 +17,8 @@
 class AccountInvoice(models.Model):
     _inherit = "account.invoice"
 
+    no_notifications = fields.Boolean(default=True)
```
Comment (ecino) [inline mobile_app_connector/models/account_invoice.py]: Please rename the field to something more understandable.
```suggestion
    avoid_mobile_donation_notification = fields.Boolean(
        default=True, help='Uncheck to send a mobile notification for donation confirmation')
```

**Review by ecino — CHANGES_REQUESTED** [review]
Please rename the field (and update all references in both pull requests).

File: mobile_app_connector/views/account_invoice_view.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_invoice_no_notifications" model="ir.ui.view">
+        <field name="name">account.invoice.attribution.form</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form" />
+        <field name="arch" type="xml">
+            <field name="returned_payment" position="after">
```
Comment (ecino) [inline mobile_app_connector/views/account_invoice_view.xml]:
```suggestion
            <field name="date_due" position="after">
```

**Review by ecino — CHANGES_REQUESTED** [review]

**Review by ecino — APPROVED** [review]

**Pull request merged**